doc/rfc/0000-rfc-repository.md

```diff
@@ -0,0 +1,168 @@
+# RFC 0000 - RFC repository
+
+- Date proposed: 2023-02-28
+- Date approved: under review
+
+## Summary
+
+This RFC proposes storing OpenTitan RFCs as files in a Git repo, and using
+an open source development workflow for the RFC process. RFCs would be
+stored in the RFC repo as text files. RFCs would be proposed, reviewed, and
+accepted/rejected using GitHub pull requests (PRs). The goal is to ensure
+project decisions are better documented, and to make the RFC process more
+visible and transparent. The workflow proposed here will already be familiar to
+open source contributors and should add little overhead.
+
+## Motivation
+
+RFCs in OpenTitan are used for decisions that need consensus from the technical
+committee. A summary of the RFC process as documented [here][rfc-process]:
+
+1. Write up your RFC somewhere.
+2. Create an issue in the OpenTitan repo with a link to your RFC.
+3. Label the issue with `RFC:Proposal`, then `RFC:TC review` when ready.
+4. Pending RFCs will be discussed in the next technical committee meeting.
+5. Approved RFCs have their label changed to `RFC:Approved`.
+6. Rejected RFCs have their label removed.
+
+Typically, authors write their RFCs in one of two places:
+
+* A Google Doc, accessible only to select people.
+* The GitHub issue itself as Markdown.
+
+RFCs stored like this are not discoverable and not portable. In the case
+of Google Docs, they are not accessible to everybody, even within the
+opentitan.org domain. RFCs are reviewed in private by the technical committee
+and project director.
+
+This makes it difficult (or impossible) for contributors to find documentation
+of what decisions have been made and why. Adopting more open processes for
+project governance will help the OpenTitan project be an exemplary case for
+open hardware development.
```

Comment:
This seems to be unnecessarily mixing concerns of discoverability and project governance. Please disentangle them and make it clear what is being proposed on both fronts. Potentially, it might even make sense to make separate proposals for those two aspects.

Reply:
Agreed, I've removed this paragraph. Changes to project governance should be separate.